### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by NARENDRA935** [learndevopsonline/learndevopsonline#1790] [Q&A]
hii,
@r-devops 
<img width="591" alt="Screenshot 2024-01-31 175718" src="https://github.com/learndevopsonline/learndevopsonline/assets/153573686/35da2894-c556-4a1c-9dda-f92acc8199ea">

**Comment by chaithanya1812** [comment]
Not a git pull, Use
git clone <url>

**Reply by NARENDRA935** [reply]
Okay 👍
